src/components/WorkAreaDock/CodeEditorComponent.tsx

```diff
@@ -39,6 +39,7 @@ export function CodeEditor(): JSX.Element {
         value={content}
         editorDidMount={onEditorMount}
         onChange={setEditorContent}
+        height="450"
```

Comment:
To żeby kod był na całym polu. Udało mi się, ale też jest brzydko zafiksowane :(